### ISSUE TITLE
Javascript bundle isn't compiling on file save

### ISSUE THREAD

**Issue by pawepaw** [madskristensen/WebEssentials2013#1844]
Created bundle with multiple files. When i edit file or edit bundle configuration (add another js file) it doesn't compile. I have to run it manually or build project.

Visual studio 2013 update 4
WE 2.5.4

```
<?xml version="1.0" encoding="utf-8"?>
<bundle xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://vswebessentials.com/schemas/v1/bundle.xsd">
  <settings>
    <!--Determines if the bundle file should be automatically optimized after creation/update.-->
    <minify>true</minify>
    <!--Determine whether to generate/re-generate this bundle on building the solution.-->
    <runOnBuild>true</runOnBuild>
    <!--Specifies a custom subfolder to save files to. By default, compiled output will be placed in the same folder and nested under the original file.-->
    <outputDirectory />
  </settings>
  <!--The order of the <file> elements determines the order of the files in the bundle.-->
  <files>
    <file>/Scripts/1.js</file>
    <file>/Scripts/2.js</file>
  </files>
</bundle>
```

**Comment by am11** [comment]
Is your project in non-System file? 
I mean is it so that your OS is installed in C: drive and project resides in other drive?
In that case, it might not work on systems. See the related discussion here: https://github.com/madskristensen/WebEssentials2013/issues/1172.

In summary, (under win 8x) .NET's `FileSystemWatcher` has issues watching files in other than OS drive.

**Comment by pawepaw** [comment]
Yes it's on different drive. So there is no real solution for this?

**Comment by am11** [comment]
Me and others whined about it on [MS Connect](https://connect.microsoft.com/VisualStudio/feedback/details/875392/microsofts-filesystemwatcher-vs-third-party-watcher-implementions-for-windows), but that was closed as "By Design".
Since `FileSystemWatcher` is now [open sourced](https://github.com/dotnet/corefx/tree/master/src/System.IO.FileSystem.Watcher), we should probably report this issue in [dotnet/corefx](https://github.com/dotnet/corefx)'s issue tracker.

> So there is no real solution for this?

Not to my knowledge.

> on different drive

Note, I have only encountered it on Windows 8.1 (not on 7). I am assuming you are also using Windows 8.1? This may imply that as if it is a bug in win32 API (as .NET eventually `P/Invoke`s the system call). But then node.js's fs.watch (and Perl, Ruby etc.) on Windows also make the same win32 API and do not exhibit this issue.

**Comment by pawepaw** [comment]
Yes it's Windows 8.1.
